packages/backend/src/accounting/service.test.ts

```diff
@@ -207,7 +207,7 @@ describe('Accounting Service', (): void => {
             timeout: 0n
           })
           assert.ok(!isTransferError(transfer))
-          await transfer.commit()
+          await transfer.post()
```

Comment:
There's some more nomenclature throughout the tests
https://github.com/interledger/rafiki/blob/04698a2d19ef442c9fc481ec005ee3a675066e57/packages/backend/src/accounting/service.test.ts#L303-L307